app/controllers/review_controller.rb

```diff
@@ -1,21 +1,37 @@
 class ReviewController < ApplicationController
   def index
     @card = current_user.card_for_review
+    respond_to do |format|
+      if @card.present?
+        format.html
+        format.json do
+          render json: @card, methods: [:image_url],
+                              except:  [:original_text, :user_id, :review_date]
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Unnecessary spacing detected.